cassandra/src/main/resources/reference.conf

```diff
@@ -5,3 +5,6 @@ freki.storage.cassandra.protocolVersion = 2
 # Default keyspace to be used in cassandra
 freki.storage.cassandra.keyspace = "freki"
 
+# Whether time series IDs should be indexed every time a new point is added (true) or if they should
+# be indexed in a more manual fashion (false).
+freki.storage.cassandra.online_index = false
```

Comment:
Maybe change this to auto_index?

Right now I do not see the connection to "online"